src/main/kotlin/se/uulm/snowballr/backend/table/ColumnHelper.kt

```diff
@@ -46,6 +46,9 @@ fun Table.deletedAt() = timestampWithTimeZone("deleted_at").nullable()
  */
 fun Table.deletedBy() = userReference("deleted_by", ReferenceOption.RESTRICT, ReferenceOption.CASCADE).nullable()
 
+/** Common column definition for the "expires at" timestamp. */
+fun Table.expiresAt() = timestampWithTimeZone("expires_at").clientDefault { OffsetDateTime.now().plusDays(1) }
```

Comment:
Would it be of interest to make this default expiry duration configurable through an environment variable?

Reply:
One day is reasonable for this use-case and configuration is currently not of importance. Resolving...